modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -1,74 +1,137 @@
 <version>1.0.0</version>
 <packaging>mule-domain</packaging>
 ----
-* You have two listeners name:
-+
-** `api-proxy-listener-http`. +
-Binds to all interfaces and uses port 8081
-** `api-proxy-listener-https`. +
-The listener configuration for HTTPS communications. +
-To use it you must uncomment code from your API Proxy template and configure your certificates and passwords.
-+
-Your API Proxy listeners should inherit your Proxy domain HTTP Listener configuration by using any of these listeners name:
-+
+. Customize the domain template to fit your usage scenarios.
+
+When referencing a domain, Mule reads the domain Maven coordinates from the API Proxy pom.xml file and looks for an already deployed domain with the same semantic name +
+For example, in the case of API gateway domain with the version specified in the previous example, the semantic name is: `gateway-proxy-domain-1.0.0-mule-domain`.
+
+For a proxy to leverage the domain, you must make sure that the domain template is specified in the API Proxy pom.xml as a provided dependency. 
+
+Example:
+
 [source,xml,linenums]
 ----
-<http:listener-config name="api-proxy-listener-http">
-  <http:listener-connection host="0.0.0.0" port="${proxy.port}" protocol="${implementation.protocol}"/>
-</http:listener-config>
+<dependency>
+    <groupId>com.mulesoft.anypoint</groupId>
+    <artifactId>gateway-proxy-domain</artifactId>
+    <version>1.0.0</version>
+    <classifier>mule-domain</classifier>
+    <scope>provided</scope>
+</dependency>
 ----
-* The domain you inherit contains a file with default properties in its `config.properties` file. This resource provides settings dynamically without the need to recompile the domain. +
-The properites in this file are:
+
+== Avoiding Port Conflicts
+
+To successfully register an API, you must deploy the API Proxy to a Mule instance using a unique endpoint URL. Automatically generated proxies use the path: `+http://0.0.0.0:8081+`. +
+
+To avoid a conflict when running multiple proxies using the same domain, make sure that the proxy paths are unique. Servers try to expose every interface, privileging the most declarative ones. For example, `local_ip` is privileged against `localhost`.
+
+Additionally, if you have multiple domains deployed on the same Mule, each listener configuration must have a unique port for all domains to be successfully deployed and available to the deployed proxies.
+
+=== API Gateway Domain template
+
+If you choose to download your API Proxy and configure it manually on your end, you may use the *API Gateway domain template*.
+
+The https://anypoint.mulesoft.com/exchange/org.mule.examples/gateway-proxy-domain/[API Gateway domain template] is configured to have a shared HTTP listener configuration ("api-proxy-listener-http") listening on the 8081 port. You can also have a shared HTTPs listener configuration ("api-proxy-listener-https"). To use the configuration, you must uncomment the code and configure the TLS context.
```

Comment:
"To use either configuration"?

Reply:
Note that in the same text below, "the TLS context" is replaced by "your certificates and passwords."

Reply:
Done! Thanks!